doc/platform_api/CMIS_and_C-CMIS_support_for_ZR.md

```diff
@@ -0,0 +1,1559 @@
+    active_apsel_hostlane3       = INTEGER                          ; active application selected code assigned to host lane 3
+    active_apsel_hostlane4       = INTEGER                          ; active application selected code assigned to host lane 4
+    active_apsel_hostlane5       = INTEGER                          ; active application selected code assigned to host lane 5
+    active_apsel_hostlane6       = INTEGER                          ; active application selected code assigned to host lane 6
+    active_apsel_hostlane7       = INTEGER                          ; active application selected code assigned to host lane 7
+    active_apsel_hostlane8       = INTEGER                          ; active application selected code assigned to host lane 8
+    media_interface_technology   = 1*255VCHAR                       ; media interface technology
+    hardware_rev                 = 1*255VCHAR                       ; module hardware revision 
+    serial                       = 1*255VCHAR                       ; module serial number 
+    manufacturer                 = 1*255VCHAR                       ; module venndor name
+    model                        = 1*255VCHAR                       ; module part number
+    vendor_rev                   = 1*255VCHAR                       ; module vendor revision
+    vendor_oui                   = 1*255VCHAR                       ; vendor organizationally unique identifier
+    vendor_date                  = 1*255VCHAR                       ; module manufacture date
+    connector                    = 1*255VCHAR                       ; connector type
+    encoding                     = 1*255VCHAR                       ; N/A
+    specification_compliance     = 1*255VCHAR                       ; module media type
+    application_advertisement    = 1*255VCHAR                       ; N/A
+    cmis_rev                     = 1*255VCHAR                       ; CMIS revision that the module complies to
+    active_firmware              = 1*255VCHAR                       ; active firmware
+    inactive_firmware            = 1*255VCHAR                       ; inactive firmware
+    supported_max_tx_power       = FLOAT                            ; support maximum tx power
+    supported_min_tx_power       = FLOAT                            ; support minimum tx power
+    supported_max_laser_freq     = FLOAT                            ; support maximum laser frequency
+    supported_min_laser_freq     = FLOAT                            ; support minimum laser frequency
+
+##### 2.1.2 Transceiver DOM sensor Table #####
+
+    ; Defines Transceiver DOM sensor information for a port
+    key                          = TRANSCEIVER_DOM_SENSOR|ifname    ; information module DOM sensors on port
+    ; field                      = value
+    tx_disable                   = BOOLEAN                          ; TX disable state
+    tx_disabled_channel          = INTEGER                          ; TX disable field
```

Comment:
It will be great if you can share the offset details for this since I wasn't directly able to find it in the spec

Reply:
tx_disable field in Page 0x10, byte 130.